base/doc/ltnews39.tex

```diff
@@ -572,30 +613,31 @@ \subsection{Declaring appropriate sub-encodings for \texttt{TS1} symbol fonts}
 Use sub-encoding 0 (not 5)
 -----------------------------------------
 \end{verbatim}
-i.e., all glyphs are provided, while in reality more than twenty are
-missing and sub-encoding 5 as declared in the kernel is correct.
+thus it claims that
+all glyphs are provided, while in reality more than twenty are missing
+and sub-encoding 5, as declared in the kernel, is in fact correct.
 %
 \githubissue{1257}
 
-\subsection{Behavior of loading \pkg{textcomp} without options}
 
-When incorporating \pkg{textcomp} package into \LaTeX{} kernel in
-release 2020-02~\cite{39:ltnews31}, the default type of package
-messages was also changed from package info (\texttt{Package textcomp Info})
+\subsection{Behavior when loading \pkg{textcomp} without options}
+
+When incorporating the \pkg{textcomp} package into the \LaTeX{} kernel, in
+the February 2020 release~\cite{39:ltnews31}, the default type of its package messages was changed from package info (\texttt{Package textcomp Info})
 to \LaTeX{} kernel info (\texttt{LaTeX Info}). But if \pkg{textcomp}
-was loaded without options, message type was restored to package info.
+was loaded without options, the message type got restored to package info.
 This restoration has now been canceled.
 
-Note that loading \pkg{textcomp} with one of options \option{error},
-\option{warn}, and \option{info} would still change message type to the
-corresponding package error, warning, or info.
+Note that loading \pkg{textcomp} with one of the options \option{error},
+\option{warn}, or \option{info} still changes the message
+type to a \pkg{textcomp} package error, warning, or info message.
```

Comment:
Is the "package" needed here? 
I still tend to read it as "package-error"! 

Maybe turn it around to this:

an error, warning or info message 
from the \pkg{textcomp} package.
OR
from \pkg{textcomp}. 
OR
from the package. 

Another, simpler, possibility 
(is any reference to the package useful here?):

to error, warning or info, as requested.

Reply:
@josephwright 

So forget the above reasoning and questions! 

I now understand that the important change is to it being the package that is in control.

So maybe better (two possibilities):

an error, warning or info message
from the \pkg{textcomp} package (rather than the kernel).  

OR

to error, warning or info, as requested (from the package rather than the kernel).